### PR TITLE
Slightly streamlined bottom part

### PR DESCRIPTION
**Pull request by karevn**
image

### CODE REVIEW

File: app/vars.scss [hunk 1]
```diff
@@ -7,7 +7,7 @@ $blue: #6272a4;
 $cyan: #8be9fd;
 $green: #50fa7b;
 $orange: #ffb86c;
-$pink: #ff79c6;
```
Comment (nukeop) [inline app/vars.scss]: Please revert this to hex for consistency

**Comment by nukeop** [comment]
Great contribution, thanks.

**Comment by karevn** [comment]
Done! :)

**Comment by nukeop** [comment]
Excellent

**Pull request merged**